framework/include/utils/MooseTypes.h

```diff
@@ -27,16 +27,11 @@
 
 #include <string>
 #include <vector>
+#include <memory>
 
-#ifdef LIBMESH_HAVE_CXX11_SHARED_PTR
-#  include <memory>
-#  define MooseSharedPointer std::shared_ptr
-#  define MooseSharedNamespace std
-#else
-#  include "boost/shared_ptr.hpp"
-#  define MooseSharedPointer boost::shared_ptr
-#  define MooseSharedNamespace boost
-#endif
+// DO NOT USE (Deprecated)
+#define MooseSharedPointer std::shared_ptr
```

Comment:
Too many instances still hanging around in modules plus this definition in MOOSE for that. The shim class will be better for supporting it with an error message for a little while, but not yet.